devtools/create_project/cmake/CMakeLists.txt

```diff
@@ -21,3 +21,5 @@ add_executable(create_project ${SOURCE_FILES})
 if (WIN32)
     target_link_libraries(create_project rpcrt4 advapi32)
 endif ()
+set_property(TARGET create_project PROPERTY CXX_STANDARD 11)
```

Comment:
description in commit doesn't reflect this, but make sense to add

Reply:
Do you want me to separate this and the Xcode to a "enable c++11" commit?